### PR TITLE
grep: add line numbers on search

### PR DESCRIPTION
**Pull request by rizqirizqi**
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

This PR adds an option for grep search to show the line numbers.
image

### CODE REVIEW

File: pages/common/grep.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 - Search for a pattern recursively in the current directory, ignoring non-text files:
```
Comment (navarroaxel) [inline pages/common/grep.md:14]: We use this format to explain which argument applies each behavior in the command.
```suggestion
- Search for a pattern [R]ecursively in the current directory, showing matching line [n]umbers, [I]gnoring non-text files:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @rizqirizqi, your contribution about show line numbers is really needed. I left a comment.

File: pages/common/grep.md (hunk 1 above)
Comment (rizqirizqi) [reply pages/common/grep.md]: Thanks!

**Review by navarroaxel — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Looks good.

**Pull request merged**